admins/pageflow/entry.rb

```diff
@@ -168,6 +180,14 @@ module Pageflow
       end
     end
 
+    collection_action :entry_types do
+      account = Pageflow::Account.find(params[:account_id])
+
+      @entry_types = helpers.entry_type_collection_for_account(account)
+
+      render(layout: false)
+    end
+
```

Comment:
This action could use a test. Do we want to prevent users from querying available entry types of accounts they are not authorized for otherwise?